### PR TITLE
Added test for sysctl path

### PR DESCRIPTION
**Pull request by jaggzh**
Some distributions don't have /sbin in the user path by default, so sysctl is not found with the plain call without the path.  This patch catches if it exists that specific path (/sbin/sysctl) and otherwise defaults to the prior method of just hoping it's found somewhere.

### CODE REVIEW

File: src/platform/unix/BuildLinuxImage.sh.in [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
 
+# Our sysctl call may be run as normal user,
+# but /sbin is not always in normal users' PATH.
+if [[ -f /sbin/sysct && -x /sbin/sysctl ]]; then sysctl_bin=/sbin/sysctl
```
Comment (kamitchell) [inline src/platform/unix/BuildLinuxImage.sh.in:5]:
```suggestion
if [[ -x /sbin/sysctl ]]; then sysctl_bin=/sbin/sysctl
```

Looked like `sysct` was a typo. `-x` covers "file exists and execute (or search) permission is granted", so just `-x` should be sufficient.

Comment (jaggzh) [reply src/platform/unix/BuildLinuxImage.sh.in:5]: > Looked like `sysct` was a typo. `-x` covers "file exists and execute (or search) permission is granted", so just `-x` should be sufficient.

-x will return true on a directory as well. -f ensures it's a file :)

Comment (jaggzh) [reply src/platform/unix/BuildLinuxImage.sh.in:5]: (I don't know how to fix the typo and update the PR)

Comment (kamitchell) [reply src/platform/unix/BuildLinuxImage.sh.in:5]: > -x will return true on a directory as well. -f ensures it's a file :)

Good point. I'd never thought of that (or seen a double test) in 40 years of programming shell scripts. TIL.

To fix it, you could just edit the file and add another commit.

**Comment by jaggzh** [comment]
Okay, updated with typo correction. :)
(I didn't think about the -x dir either -- someone in #bash in irc brought it up).

**Comment by supermerill** [comment]
merged for next verison

**Pull request closed without merging**